packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -272,9 +273,21 @@ export const useSliderState = (state: SliderState) => {
       }
     } else if (Array.isArray(marks) && marks.length > 0) {
       // 2. We receive an array with numbers: mark for every value in array.
```

Comment:
This comment has been moved to line 279 so it should probably be removed from here.